### PR TITLE
[FSDP][Loss] Switch gradient reduction to SUM

### PR DESCRIPTION
**Pull request by HAOCHENYE**
**This PR (3/5)** — the single atomic semantic flip. FSDP reduce-scatters with pure SUM; `scale_and_reduce_grad` drops the expert `div_(ep_size)` and replicated `div_(flat_mesh.size())`; CE drops the WORLD autograd all_reduce; balancing uses `local_gating_sum` with global detached stats; z-loss drops `x world_size`; compose/VLM `fully_shard` also wired to SUM. Carries all regression tests (bf16 mechanism, token-mean parity EP1/EP2, balancing & z-loss new-vs-old A/B, compose hook).

Full stack (merge bottom-up, under #1959):
1. `reduce-sum-01-fsdp-helper` — [FSDP] Add reduce-sum gradient reduction helper
2. `reduce-sum-02-split-logging` — [Loss] Split logging loss from backward loss
3. `reduce-sum-03-switch-sum` — [FSDP][Loss] Switch gradient reduction to SUM
4. `reduce-sum-04-remove-world-size` — [Refactor] Remove unused world_size plumbing
5. `reduce-sum-05-drop-nonglobal` — [Refactor] Drop non-global aux-loss averaging mode

### CODE REVIEW

File: xtuner/v1/loss/moe_loss.py [hunk 1]
```diff
@@ -298,20 +294,18 @@ def accumulate(
...
             denom_global = torch.clamp(num_tokens_global, min=1)
-            local_loss = base * num_tokens_local / denom_global
-            loss = local_loss * world_size
+            loss = base * num_tokens_local / denom_global
 
-        local_loss = local_loss * self.loss_cfg.z_loss_alpha / self._batch_size
         loss = loss * self.loss_cfg.z_loss_alpha / self._batch_size
         self._update_running(loss.detach())
-        self._update_local_running(local_loss.detach())
+        self._update_local_running(loss.detach())
```
Comment (HAOCHENYE) [inline xtuner/v1/loss/moe_loss.py:308]: If the changes in PR 1962 are necessary, it is recommended to merge PR 1962 and PR 1963 into a single one, as it makes no sense for a function defined in PR 1962 to be called in PR 1963 — it feels fragmented

File: xtuner/v1/model/base.py [hunk 2]
```diff
@@ -1388,6 +1392,21 @@ def post_micro_batch_forward(self, batch_outputs: Sequence[ModelOutputs]) -> Bat
         if "reduced_base_loss" in reduced_other_losses:
             reduced_other_losses["reduced_llm_loss"] = reduced_other_losses.pop("reduced_base_loss")
 
+        # Safety net: every `*loss` tensor field an output exposes must have produced a display curve
+        # from a registered local component. Otherwise a newly added loss term would silently vanish
+        # from the logged curves while still driving backward, hiding the regression.
+        for output in batch_outputs:
```
Comment (HAOCHENYE) [inline xtuner/v1/model/base.py:1398]: Encapsulate it into an internal function; subclasses can override it if needed

**Comment by HAOCHENYE** [comment]
Superseded by the consolidated 2-PR stack #1968 (core) + #1969 (drop non-global). Closing this fragmented stack.
